src/main/common/sensor_alignment.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "axis.h"
+#include "maths.h"
+
+typedef enum {
+    ALIGN_DEFAULT = 0, // driver-provided alignment
+
+    // the order of these 8 values correlates to corresponding code in ALIGNMENT_TO_BITMASK below.
+
+                            // R, P, Y
+    CW0_DEG = 1,            // 00,00,00
+    CW90_DEG = 2,           // 00,00,01
+    CW180_DEG = 3,          // 00,00,10
+    CW270_DEG = 4,          // 00,00,11
+    CW0_DEG_FLIP = 5,       // 00,10,00 // _FLIP = 2x90 degree PITCH rotations
+    CW90_DEG_FLIP = 6,      // 00,10,01
+    CW180_DEG_FLIP = 7,     // 00,10,10
+    CW270_DEG_FLIP = 8,     // 00,10,11
+
+    // Note: avoid using enum values which could be useful for the alignment bitmask code, i.e values 1 + 000000b (1), to 1 + 111111b (65)
+
+    ALIGN_CUSTOM = 128,    // arbitrary sensor angles, e.g. for external sensors
```

Comment:
This should be 9.

Reply:
ahh yes as its synced with the settings.c tables, will fix.